Uplift/Controllers/SportsFormViewController.swift

```diff
@@ -0,0 +1,265 @@
+//
+//  SportsFormViewController.swift
+//  Uplift
+//
+//  Created by Artesia Ko on 5/23/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol SportsFormBubbleListDelegate: class {
+    func didTapDropdownHeader(identifier: String)
+}
+
+class BubbleItem {
+    var title: String
+    var isSelected: Bool
+    
+    init(title: String, isSelected: Bool) {
+        self.title = title
+        self.isSelected = isSelected
+    }
+}
+
+class SportsFormViewController: UIViewController {
+     
+    private var cancelButton = UIButton()
+    private var createButton = UIButton()
+    private var headerLabel = UILabel()
+    private var headerRect = UIView()
+    
+    private var collectionView: UICollectionView!
+    
+    private struct Section {
+        var items: [ItemType]
+    }
+    
+    private enum ItemType {
+        case name
+        case time
+        case sport([BubbleItem])
+        case location([BubbleItem])
+        case players
+    }
+    
+    private var sportsDropStatus: DropdownStatus = .closed
+    private var locationDropStatus: DropdownStatus = .closed
+    private var section: Section!
```

Comment:
can we move the `super.init(nibName: nil, bundle: nil)` to the end of the init function and then drop the `!` here? ideally avoid using `!` whenever possible